app/views/compute_resources/show.html.erb

```diff
@@ -45,7 +45,7 @@
   </div>
   <% if @compute_resource.capabilities.include?(:image) %>
     <div id="images" class="tab-pane">
-      <%= title_actions(display_link_if_authorized(_("New Image"), hash_for_new_compute_resource_image_path(:compute_resource_id => @compute_resource.id), :class => "btn btn-success", :data=>{:target=>"full-page"})) %>
+      <%= title_actions(display_link_if_authorized(_("New Image"), hash_for_new_compute_resource_image_path(:compute_resource_id => @compute_resource.id), :class => "btn btn-primary", :data=>{:target=>"full-page"})) %>
```

Comment:
It'd be good if `new_link` could help here too, perhaps by passing an optional hash for the link which is merged with the `{:action => 'new'}`?